src/Forms/Prism.Forms/Ioc/ContainerRegistryExtensions.cs

```diff
@@ -6,7 +6,10 @@
 
 namespace Prism.Ioc
 {
-    public static class IContainerRegistryExtensions
+    /// <summary>
+    /// Extension methods for registering containers such as pages for navigation, dialog, etc.
+    /// </summary>
+    public static class ContainerRegistryExtensions
```

Comment:
Do not rename our classes

Reply:
Just to be clear here, the renaming was because a class, not an interface, was following the interface standard for naming conventions, e.g. IClassName. Are you saying there is a preference here for that? If so, how do you distinguish what can be a class vs. an interface when navigating the directory? Likewise, how do you know what class should be named with the I convention, and what should not (asking for if I were to start adding more to the library, so that I am aware).

Reply:
This is an extension class specifically for the IContainerRegistry interface. Therefore it is named to indicate which interface the extensions are written for. It also places the file directly next to the interface in visual studio solution explorer so it is much easier to find.

Reply:
Makes sense, thank you for the clarification.

Reply:
Done